test/parallel/test-domain-error-types.js

```diff
@@ -6,10 +6,17 @@ const assert = require('assert');
 const domain = require('domain');
 
 // This test is similar to test-domain-multiple-errors, but uses a new domain
-// for each errors.
+// for each error.
 // The test flags are not essential, but serve as a way to verify that
 // https://github.com/nodejs/node/issues/28275 is fixed in debug mode.
 
+// Unfortunately, the test crashes frequently in CI in the debug + container +
```

Comment:
Could you perhaps add a `TODO` line to make such mention easier to find?

Reply:
Moved to the status file. but if it ends up that the skip code does need to be in the JS file, then yes, definitely will add a TODO.

Reply:
Moved to the status file. but if it ends up that the skip code does need to be in the JS file, then yes, definitely will add a TODO.